src/mmw/js/src/modeling/controllers.js

```diff
@@ -41,42 +43,82 @@ var ModelingController = {
                         project.get('scenarios').makeFirstScenarioActive();
                     }
                     project.getResultsIfNeeded();
+
+                    // If this project is an activity then the application's
+                    // behaior changes but only if the project is owned by the
+                    // current user.
+                    if (App.user.loggedInUserMatch(project.get('user_id')) && project.get('is_activity')) {
```

Comment:
We cannot use `App.user.loggedInUserMatch` here because the `App.user` isn't initialized until later. In the following example, I revisit my own project made using `itsi_embed=true` in non-embedded mode, and even though it is an activity and I own it, `activityMode` is `false`:

![image](https://cloud.githubusercontent.com/assets/1430060/8943288/1258c836-3548-11e5-838b-e46a6aea0d06.png)